apps/catalogue/src/components/detailView/OntologyFieldValue.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <ObjectDisplay
+    :class="'font-weight-bold mr-2 mb-2 badge bade-lg badge-' + color"
+    :data="data"
+    :metaData="metaData"
+  />
+</template>
+
+<script>
+import { ObjectDisplay } from "@mswertz/emx2-styleguide";
```

Comment:
You could here perfectly make use of vue inheritance. Why don't you? 

Secondly, should we not move this component to styleguide module, because it is basically the mirror of the *Input components there.

Reply:
"not move this component to styleguide" yes , we should but wanted to wait to discuss differences ( if any) between rendering values en a table cell of read only card , or other context .. , propose to have a separate PR as there may be some edge cases  

'vue inheritance' makes it hard to see what is going on an avoid state bugs, suggest using shared lib and have explicit component api via props